### PR TITLE
Add driver home screen

### PR DESCRIPTION
**Pull request by mayeverest**

### CODE REVIEW

File: client/routes/index.js [hunk 1]
```diff
@@ -8,21 +8,22 @@ import RegisterInfo from './RegisterInfo';
...
-
+import DriverHome from './DriverHome';
 
 const Stack = createNativeStackNavigator();
 
 function RMRoutes() {
   return (
     <NavigationContainer>
-      <Stack.Navigator initialRouteName='Landing'>
+      <Stack.Navigator initialRouteName='DriverHome'>
```
Comment (avannus) [inline client/routes/index.js]:
```suggestion
      <Stack.Navigator initialRouteName='Landing'>
```

**Review by avannus — CHANGES_REQUESTED** [review]
Looks great, you're gonna have to teach me how you did all this :^)

**Review by avannus — APPROVED** [review]

**Review by avannus — APPROVED** [review]
Updated to reset the initial route back :)

**Review by anaik62 — APPROVED** [review]
looks good!

**Review by jennyliu7 — APPROVED** [review]
Looks good to me!

**Pull request merged**